changelogs/fragments/3296-clean-etag.yaml

```diff
@@ -0,0 +1,2 @@
+minor_changes:
+  - "redfish_command and redfish_config and redfish_utils module utils - add parameter to strip etag of quotes before patch, since some vendors do not properly If-Match etag with quotes (https://github.com/ansible-collections/community.general/pull/3296)."
```

Comment:
```suggestion
  - "redfish_command and redfish_config and redfish_utils module utils - add parameter to strip etag of quotes before patch, since some vendors do not properly ``If-Match`` etag with quotes (https://github.com/ansible-collections/community.general/pull/3296)."
```